src/main/java/org/sunbird/assessment/service/AssessmentUtilServiceImpl.java

```diff
@@ -213,4 +218,25 @@
 	public Map<String, Object> getAnswerKeyForAssessmentAuthoringPreview(Map<String, Object> contentMeta) {
 		return null;
 	}
+
+	/**
+	 * To remove answers from the assessment question sets
+	 * 
+	 * @param assessmentContent
+	 *            Map<String, Object>
+	 * @return QuestionSet
+	 */
+	@Override
+	public QuestionSet removeAssessmentAnsKey(Map<String, Object> assessmentContent) {
+		QuestionSet questionSet = new ObjectMapper().convertValue(assessmentContent, QuestionSet.class);
+		List<String> qnsTypes = Arrays.asList(MCQ_MCA, MCQ_MCA, "fitb", "mtf");
```

Comment:
Use Constants for "fitb", "mtf" in Constants file.
I guess you are missing MCQ_SCA